enhancements/logging/20230130-use-otel-for-logging

```diff
@@ -0,0 +1,82 @@
+# Use OTEL for logging
+
+## Summary: 
+Currently logs are collected by fluentbit, sent to fluentd, sent to data prepper, then finally sent to Opensearch where preprocessing is done in an ingest pipeline.  I propose that we collect the logs using OTEL collector as an agent and then sent to a local OTEL collector.  The logs will then be sent through the agent using the OTLP and forwarded to a central pipeline for preprocessing.
+
+## Use case: 
+This will simplify the logging pipeline and allow use to consolidate on using OTEL collector for multiple verticles
+
+## Benefits: 
+* Improve maintainbilty my moving preprocessing out of an internal java project to a separate service
+* Simplify sending logs via the Agent as OTLP is a gRPC format
+* Align with the Open Telemetry project for logging
+* We don't expose the Opensearch API outside of the central cluster so we never have to transmit credentials, or worry about escalated access to the API.
+
+## Impact: 
+* Remove dependence on the Banzaicloud operator. Fluentbit and Fluentd would be removed. However this means we will need to build our own log aggregation controller.
+* This also means we will diverge from rancher-logging in its current form.
+* We will need a migration path to deprecate the ingest plugin.
```

Comment:
What would the migration plan be for users upgrading from the old logging capability to this one? This needs to be included here at a high level at minimum

Reply:
@sanjay920 I've added details to the Implementation section and Acceptance Criteria that detail the migration path